cclient/request/broadcast_transactions.h

```diff
@@ -16,12 +16,12 @@ extern "C" {
 
 typedef struct broadcast_transactions_req_s {
   // List of raw transactions data to be broadcasted
-  flex_hash_array_t* trytes;
+  hash8019_queue_t trytes;
```

Comment:
I think order does not matter for this call, you can use a stack for lower memory impact.